lib/vega_lite/data.ex

```diff
@@ -403,42 +403,58 @@ defmodule VegaLite.Data do
         legend_key: legend_key
       )
   """
-  def polar_plot(vl \\ Vl.new(), data, fields, opts \\ []) do
+  def polar_grid(vl \\ Vl.new(), opts \\ []) do
```

Comment:
I am not sure this signature is correct. I think you want to break it apart such data `polar_grid(opts \\ [])` and then `polar_grid(vl, opts)`. In other words, we want the arity 1 to receive opts, no the VL.

Reply:
Fixed